packages/skill-lesson/markdown/transcript-components.tsx

```diff
@@ -18,39 +19,45 @@ const getTranscriptComponents = ({
 }) => {
   return {
     p: ({children}: any) => {
-      const text = children.toString()
-      const timestampRegex = /(\d+:\d+)/
-      const matches = text.match(timestampRegex)
-
-      if (matches) {
-        const timestamp = matches[1]
-        const beforeText = text.split(matches[0])[0]
-        const afterText = text.split(matches[0])[1]
-        return (
-          <p>
-            {beforeText.replace('[', '')}
-            {canShowVideo ? (
-              <button
-                data-timestamp=""
-                className="after:content-[' '] inline-block underline after:inline-block"
-                onClick={() => {
-                  if (muxPlayerRef.current) {
-                    muxPlayerRef.current.currentTime = hmsToSeconds(timestamp)
-                    handlePlay()
-                    window.scrollTo({top: 0})
-                  }
-                }}
-              >
-                {timestamp}
-              </button>
-            ) : (
-              timestamp
-            )}
-            {afterText.replace(']', '')}
-          </p>
-        )
-      }
-      return <p>{text}</p>
+      const elements = React.Children.toArray(children)
+      const updatedChildren = elements.map((child, index) => {
+        if (typeof child === 'string') {
```

Comment:
tagging @vojtaholik to review this PR as well because I'm not as familiar with this `transcript-component` and it is used across several apps